lms/djangoapps/bulk_email/tests/test_tasks.py

```diff
@@ -278,6 +278,36 @@ def test_ses_domain_ends_with_dot(self):
         # Test that celery handles permanent SMTPDataErrors by failing and not retrying.
         self._test_email_address_failures(SESDomainEndsWithDotError(554, "Email address ends with a dot"))
 
+    def test_email_address_with_ascii_characters_failed(self):
+        """
+        Tests that bulk email skips the email address containing non-ASCII characters
+        and does not fail.
+        """
+        num_emails = 10
+        emails_with_non_ascii_chars = 3
+        num_of_course_instructors = 1
+
+        students = [self.create_student('robot%d' % i) for i in range(num_emails)]
+        for index, student in enumerate(students, 1):
```

Comment:
I think we need to make it more simpler, may be something without `enumerate` and `break` statement.

Reply:
Can you please suggest any other way?

Reply:
What about?
``` python
for student in students[:emails_with_non_ascii_chars]:
     student.email = '{username}@tesá.com'.format(username=student.username)
     student.save()
````